### ISSUE TITLE
Full Site Editing: TemplateUpdateConfirmationButton causes rendering loop

### ISSUE THREAD

**Issue by glendaviesnz** [Automattic/wp-calypso#34705]
#### Steps to reproduce
1. Check out branch follow instructions in https://github.com/Automattic/wp-calypso/tree/master/apps/full-site-editing#build-system. Install the FSE plugin, and activate it on local gutenberg dev
2. Add console log to /full-site-editing/full-site-editing-plugin/full-site-editing/plugins/template-update-confirmation/button.js componentDidUpdate
3. Edit a template part, eg. Header or Footer with console open - console.log will show the component constantly rerendering
4.  Comment the following in componentDidUpdate and re-render stops - so looks like the onResize method is causing the render loop somehow.

```
	if ( props.isFullScreen !== prevProps.isFullScreen ) {
			// this ensures the button is repositioned properly when toggling fullscreen mode
			setTimeout( () => this.onResize(), 1 );
		}
```

**Comment by glendaviesnz** [comment]
Start of a potential fix here - https://github.com/Automattic/wp-calypso/commit/9634d20a8764c8f3ce4b08ee09123d272466d9ae rendering the button in the toolbar instead of absolute positioning removes the need to watch for full screen editing changes

**Comment by glendaviesnz** [comment]
This should be fixed if we go forward with #34728 as the confirmation panel button will be removed
